README.md

```diff
@@ -1,21 +1,24 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/forseti-security.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/forseti-security) [![codecov](https://codecov.io/gh/GoogleCloudPlatform/forseti-security/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/forseti-security)
 
 # Forseti Security
-
 A community-driven collection of open source tools to improve the security
 of your Google Cloud Platform environments.
 
 [Get Started](http://forsetisecurity.org/docs/quickstarts/forseti-security/)
 with Forseti Security.
 
 ## Contributing
-
 We are continually improving Forseti Security and invite you to submit feature
 requests and bug reports under Issues. If you would like to contribute to our
 development efforts, please review our
 [contributing guidelines](/.github/CONTRIBUTING.md) and submit a pull request.
 
-## Community
+### forsetisecurity.org
+If you would like to contribute to forsetisecurity.org the website and its
+content is contained in the `gh-pages` branch. Visit its
+[README](https://github.com/GoogleCloudPlatform/forseti-security/tree/gh-pages)
```

Comment:
This link doesn't directly open the README page.  Do you want it to?

Reply:
Nope. I thought linking to the branch where the README is rendered below was best.